addon/templates/components/welcome-page.hbs

```diff
@@ -1,10 +1,10 @@
-<div id="ember-welcome-page-id-selector" data-ember-version="{{emberVersion}}">
+<main id="ember-welcome-page-id-selector" data-ember-version="{{emberVersion}}">
```

Comment:
From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main:

> A document musn't have more than one <main> element that doesn't have the hidden attribute specified.

Should we steer folks toward better HTML defaults by putting this `<main>` tag in our main app blueprint instead? If we put a `main` here we handle accessibility within this addon but don’t resolve the larger initial accessibility setup for new apps ...

Reply:
> Should we steer folks toward better HTML defaults by putting this <main> tag in our main app blueprint instead?

Yes, I think we should address this elsewhere.

Reply:
I would then recommend that we put the main element in `application.hbs`, surrounding the invocation of this component. That way it is visible to the end-programmer and we don't force a main element on the application.